tools/manifest/sourcefile.py

```diff
@@ -1016,18 +1016,37 @@ def manifest_items(self) -> Tuple[Text, List[ManifestItem]]:
                 ))
 
         elif self.content_is_ref_node:
-            rv = RefTest.item_type, [
-                RefTest(
-                    self.tests_root,
-                    self.rel_path,
-                    self.url_base,
-                    self.rel_url,
-                    references=self.references,
-                    timeout=self.timeout,
-                    viewport_size=self.viewport_size,
-                    dpi=self.dpi,
-                    fuzzy=self.fuzzy
-                )]
+            if self.test_variants:
+                rv = RefTest.item_type, []
+                for variant in self.test_variants:
```

Comment:
I don't think we ever hit the `else` clause, because `test_variants` seems to always return at least the empty variant.